### DISCUSSION TITLE
[General] <Title> Issue Title: Incorrect USDT Amount Conversion in zkSync CLI Documentation

### DISCUSSION THREAD

**Discussion by samui68400** [zkSync-Community-Hub/zksync-developers#540] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Description:

**When using the zkSync CLI for token transfers, the conversion for ERC20 tokens amounts is not clearly documented.** 

For example with USDT token contract 0x8C9d66bA3E1D7681cfFFfa3C7d9807adae368E74 on testnet

command:

`npx` zksync-cli wallet transfer --token `0x8C9d66bA3E1D7681cfFFfa3C7d9807adae368E74`

**results in an unexpected conversion rate. For instance, entering 0.0000000001 as the amount corresponds to 100 USDT, which is not intuitive and not documented. This issue leads to confusion and potential errors in transactions.**

### Additional Details

**Example Transaction:**

The following transaction demonstrates the issue: https://sepolia.explorer.zksync.io/tx/0x96a464f74b7990f167fb47383fce283be24e27b5a2bcf885beaa6670c566001c

**Steps to Reproduce:**

Use the zkSync CLI to transfer USDT.
Specify the amount as 0.0000000001.
Observe that the transferred amount is 100 USDT.

**Expected Behavior:**

The documentation should provide clear guidelines on how to specify the amount for token transfers with non-standard decimals.

**Environment:**

zkSync CLI Version: [Specify Version]
Network: Sepolia Testnet

**Additional Context:**

This issue can lead to incorrect transaction amounts and should be addressed to ensure accurate and user-friendly documentation.

**Comment by bxpana** [comment]
Thanks for posting, will look into this.

**Reply by samui68400** [reply]
thank you, I'll be waiting for the answer

**Reply by samui68400** [reply]
Any news?

**Reply by bxpana** [reply, marked as answer]
Sorry about the wait here. This should be working now after this [PR](https://github.com/matter-labs/zksync-cli/pull/149). 

Let us know if you're still having issues with it
